blog/_posts/2021-10-08-old-school-remote-dev-clusters.md

```diff
@@ -0,0 +1,163 @@
+---
+slug: "old-school-remote-dev-clusters"
+date: 2021-10-07
+author: nick
+layout: blog
+title: "Setting up a Remote Dev Environment When You Hate the Cloud"
+subtitle: "We'll use Tailscale and Kind to Duct Tape Together a Dev Env"
+description: "We'll use Tailscale and Kind to Duct Tape Together a Dev Env"
+image: "/assets/images/old-school-remote-dev-clusters/cloud.jpg"
+tags:
+  - ctlptl
+  - tailscale
+  - kind
+---
+
+Lately, when I try to present demo apps, they're often very slow. 
+Zoom is a CPU hog. And doesn't leave much CPU left for my apps to run.
+When I try to run demos on MacOS with Docker Desktop running, it's even
+worse.
+
+There are lots of teams who are trying to sell you on a remote
+dev environment as a service. And that's fine. This is a good problem
+to solve. Maybe we'll all be on dev envs as a service some day.
+
+But we live in a world where there are computers EVERYWHERE.  In my
+pocket. Under my desk. I wondered: could I offload my dev environment to ANY
+random computer?
+
+It turns out you can!
+
+In this post, I want to share how I set up my standard remote dev environment.
+
+We'll use:
+
+- A box under my desk.
+
+- [Tailscale](https://tailscale.com/) to create a secure tunnel to the box.
+
+- A Docker daemon that listens on the Tailscale network.
+
+- [Kind](https://kind.sigs.k8s.io/) to create a self-contained Kubernetes environment.
+
+Let me show you!
+
+## Setting up Tailscale
+
+Let's be clear: we're setting up a remote box that can run arbitrary servers.
+
+If we don't want it to be p0wned by crypto miners, we'll need to secure it.
```

Comment:
> p0wned

Nick.

Reply:
at least remove the 0, this spelling doesn't even make sense 😬